rotkehlchen/globaldb/updates.py

```diff
@@ -138,7 +136,7 @@ class AssetsUpdater():
     def __init__(self, msg_aggregator: MessagesAggregator) -> None:
         self.msg_aggregator = msg_aggregator
         self.local_assets_version = GlobalDBHandler().get_setting_value(ASSETS_VERSION_KEY, 0)
-        self.last_remote_checked_version = None
+        self.last_remote_checked_version = -1  # integer value that represents no update
```

Comment:
I changed it to avoid the weird ignore type comments since we are doing arithmetical operations with a nullable value. Changed it to -1 since it won't break the logic

Reply:
Alright :+1: But then a comment on line 659 should be removed